.github/workflows/hyperion_java-client_publish.yml

```
@@ -0,0 +1,115 @@
+        description: 'Force publish (even on PR)'
+        required: false
+        default: 'false'
+
+jobs:
+  validate:
+    name: Validate and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)